html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <form action="http://www.example.com">
+    <h1>Coding Form!</h1>
+    <div>
+      Email
```

Comment:
Please wrap the titles for your form elements in label elements ! For example: 
< label for="email-address" > Email </ label >